app/src/main/res/values/strings.xml

```diff
@@ -172,4 +172,6 @@
     <string name="cd_navdrawer_user_picture">User Profile Image</string>
     <string name="title_signing_needed">Sign-in Needed</string>
     <string name="signin">Sign-in</string>
+    <string name="feedback_message_required">Message required</string>
+    <string name="feedback_invalid_email">Input valid email address</string>

```

Comment:
BTW, what should we do here? It's just a single string change (for english) but we have localizations implemented so would require string values for other languages.